pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -19,7 +19,7 @@
 , cephSupport ? false, ceph
 , openGLSupport ? sdlSupport, mesa, epoxy, libdrm
 , virglSupport ? openGLSupport, virglrenderer
-, smbdSupport ? false, samba
+, smbdSupport ? true, samba
```

Comment:
This should not be unconditional as it pull in samba as a dependency. Set it to `!nixosTestRunner` instead if you must.

Instead of doing this I'd rather have a `qemu_full` package that enables all the optional features. Things like `cephSupport` are probably more or just as likely to be used as samba.

Reply:
Good idea. I did not realize how big `samba` package is. Is it going to be based on `qemu` or `qemu_kvm` ?

Reply:
I think they are the same package and if not: I would use qemu_kvm because full.

Reply:
`qemu` is the fuller one, so I used that and updated the pull request.